resources/saas-boost-svc-onboarding.yaml

```diff
@@ -564,6 +572,16 @@ Resources:
                   - ec2:AuthorizeSecurityGroupEgress
                 Resource:
                   - !Sub arn:aws:ec2:${AWS::Region}:${AWS::AccountId}:security-group/*
+              # Elasticsearch onboarding policy
+              - Effect: Allow
+                Action:
+                  - es:AddTags 
+                  - es:CreateElasticsearchDomain  
+                  - es:DescribeElasticsearchDomain
+                  - es:CreateElasticsearchServiceRole
+                Resource: 
+                  - !Sub arn:aws:es:${AWS::Region}:${AWS::AccountId}:domain/*
```

Comment:
In keeping with the principle of least privilege, it would be better if our Elasticsearch policies only gave us access to the domain managed by SaaS Boost, not all Elasticsearch domains in that region for that AWS account id.